ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/siteConfig/siteConfigBuilder.go

```diff
@@ -81,6 +81,7 @@ func (scbuilder *SiteConfigBuilder) getClusterCR(clusterId int, siteConfigTemp S
 	// Adding extra manifest
 	if mapSourceCR["kind"] == "ConfigMap" {
```

Comment:
Does this make an assumption that there is only one ConfigMap in the source CRs? If so we need to document that. If not this should be tightened up to replace the data in only the correct ConfigMap.

Reply:
yes, it is one configMap that exist in the installation CRs